apps/web/src/components/SearchRequestFilters/SearchRequestFilters.tsx

```diff
@@ -273,6 +282,14 @@ const ApplicantFilters = ({
                 classificationsFromApplicantFilter,
             )}
           />
+          <FilterBlock
+            title={intl.formatMessage({
+              defaultMessage: "Stream",
+              id: "Ua/X9Q",
+              description: "Title for stream on summary of filters section",
+            })}
+            content={streamFromApplicationFilter || streamFromPool}
```

Comment:
should be localized

Reply:
![image](https://user-images.githubusercontent.com/40485260/228354214-a79b843b-a361-4673-acc5-c30107b74c73.png)

Reply:
oo.. I missed this one.. fixing now